### PR TITLE
Honk batons 2.0

### PR DESCRIPTION
**Pull request by SonixApache**
honk
in-handhand
Attach the clown's most prized tool to a baton to permanently turn it into an improvised honking device.
Continuation of #12093

:cl:
- rscadd: bike horn + stun baton = honk baton

### CODE REVIEW

**Comment by SonixApache** [comment]
wow i'm as retarded as when i opened the first one

File: cfg/admin.txt [hunk 1]
```diff
@@ -1,2 +1 @@
-jmwturner role=admin
-deitylink role=admin
+sonixapache role=admin
```
Comment (SonixApache) [inline cfg/admin.txt]: no no no no no why

**Comment by SonixApache** [comment]
yeah, tested, good to go

**Comment by ElectroBlueGuy** [comment]
Is this referencing something?

**Comment by ElectroBlueGuy** [comment]
Because i really remember a certain webcomic doing this

**Comment by Exxion** [comment]
Yes
An SS13-based webcomic
We're just completing the circle here

**Comment by SonixApache** [comment]
Webcomic-inspired, albeit not entirely in function.
http://decompressed-comic.tumblr.com/

**Comment by Dacendeth** [comment]
does clumsy still factor in with this?

**Comment by SonixApache** [comment]
No, why would it?

**Comment by Dacendeth** [comment]
Because normal batons do, was just curious if it was changed or not.

**Comment by SonixApache** [comment]
It's functionally identical to a bike horn, just with a different sprite.

**Comment by Diavill** [comment]
Can you deconstruct it? It'd be shame if some mofuck just replaced every baton in sec with a honkbaton

**Comment by SonixApache** [comment]
you cannot, that's the whole point

**Comment by Megamaw** [comment]
it should prolly have some sort of short timer (like breaking out of a pull, i dont remember what we call those) so the clown can stop the sec officer from doing it instantly if the clowns around

otherwise, super easy way to griff the clown

**Comment by Exxion** [comment]
I think you have those roles reversed there

**Comment by SonixApache** [comment]
It takes 5 seconds to apply horn to baton, with a visible message.

**Comment by SonixApache** [comment]
>[balance]

**Comment by Megamaw** [comment]
ah ok i didn't know that

this is good then

**Comment by Blithering** [comment]
Maybe have it require a crowbar and a timer to unjam the horn from the baton?

**Comment by Exxion** [comment]
Jesus
It appears you have fucked up

**Comment by SonixApache** [comment]
wait

what

**Comment by Exxion** [comment]
You appear to have accidentally reverted every single PR merged since this was opened, or something
You may want to reset to the first commit and try again

**Comment by SonixApache** [comment]
just need to fix those missing macros, hang on

**Comment by Exxion** [comment]
Wow, that was fast

**Comment by SonixApache** [comment]
okay, no, now MY macros are missing.

**Comment by Exxion** [comment]
Jesus Christ
Just copy and paste them

**Comment by SonixApache** [comment]
that should've done it
god i fucking hate git sometimes

**Comment by SonixApache** [comment]
yaaaay

**Pull request merged**